docs/src/logging.md

```diff
@@ -1,43 +1,50 @@
 # Logging
 
-The operator is designed to log in JSON format directly to standard output,
-including PostgreSQL logs.
+The operator outputs logs in JSON format directly to standard output, including
+PostgreSQL logs, without persisting them to storage for security reasons. This
+design facilitates seamless integration with most Kubernetes-compatible log
+management tools, including command line ones like
+[stern](https://github.com/stern/stern).
 
-Each log entry has the following fields:
+!!! Important
+    Long-term storage and management of logs are outside the scope of the
+    operator and should be handled at the Kubernetes infrastructure level.
+    For more information, see the
+    [Kubernetes Logging Architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/)
+    documentation.
 
-- `level` – Log level (`info`, `notice`, ...).
+Each log entry includes the following fields:
+
+- `level` – The log level (e.g., `info`, `notice`).
 - `ts` – The timestamp (epoch with microseconds).
```

Comment:
I've been trying to get a definitive answer on the timestamp format. I cannot find one. Different loggers use unix epoch or RFC3390, for example `zap` uses the latter. I'd probably remove the parenthesized part here.

Reply:
Huh, so the timestamp has recently been fixed to unix epoch here.
https://github.com/cloudnative-pg/machinery/pull/38/files
I will put this back in.

Reply:
Ah, no! We're fixing to nanoseconds!
You know what, I think we don't need to specify the format in this doc.